src/main/markdown/gettingstarted-v2.md

```diff
@@ -44,6 +44,16 @@ mvn archetype:generate \
    -DarchetypeArtifactId=modular-webapp
 ```
 
+If you are using a Java version earlier than 17, consider using a previous Maven archetype. The recent upgrade of Jetty to version 12 
+requires Java 17 or newer.
+
+```shell
+mvn archetype:generate \
+   -DarchetypeGroupId=net.ltgt.gwt.archetypes \
+   -DarchetypeVersion=2024.11.13 \
+   -DarchetypeArtifactId=modular-webapp
+```
+
```

Comment:
Good point. 

may be mentioning, that the new version can be used and the Java version can be updated manually inside the pom and a hint that future versions of GWT will require Java 17. So, it is a good idea doing a fresh start with a new project and Java 17.